lib/terrapin/command_line/multi_pipe.rb

```diff
@@ -47,8 +47,8 @@ def close_read
       end
 
       def read_streams(output, error)
-        @stdout_output = ""
-        @stderr_output = ""
+        @stdout_output = String.new
+        @stderr_output = String.new
```

Comment:
```suggestion
        @stdout_output = +""
        @stderr_output = +""
```

This fix will also work in Ruby 2.2 or higher.